app/models/user.py

```diff
@@ -57,10 +57,11 @@ class User(UserMixin, db.Model):
     def __init__(self, **kwargs):
         super(User, self).__init__(**kwargs)
         if self.role is None:
```

Comment:
[StackOverflow](http://stackoverflow.com/questions/20460339/flask-sqlalchemy-constructor) says it's a sqlalchemy thing. The `User` model inherits `db.Model`. That base `db.Model` loops through the kwargs and assigns fields to them.